### PR TITLE
Switch to bevy 0.11

### PR DESCRIPTION
**Pull request by alepez**

### CODE REVIEW

**Comment by gschup** [comment]
Thanks for doing the work :)

**Comment by alepez** [comment]
Sorry, tests were still broken. Now they should be fixed.

File: examples/box_game/box_game_spectator.rs [hunk 1]
```diff
@@ -62,8 +62,8 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
             2.0,
             TimerMode::Repeating,
         )))
-        .add_system(print_network_stats_system)
-        .add_system(print_events_system)
+        .add_systems(Startup, print_network_stats_system)
+        .add_systems(Startup, print_events_system)
```
Comment (simbleau) [inline examples/box_game/box_game_spectator.rs:66]: Isn't this supposed to be Update, not Startup?

File: examples/box_game/box_game_spectator.rs [hunk 2]
```diff
@@ -62,8 +62,8 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
             2.0,
             TimerMode::Repeating,
         )))
-        .add_system(print_network_stats_system)
-        .add_system(print_events_system)
+        .add_systems(Startup, print_network_stats_system)
```
Comment (simbleau) [inline examples/box_game/box_game_spectator.rs:65]: Isn't this supposed to be Update, not Startup?

File: examples/box_game/box_game_spectator.rs (hunk 1 above)
Comment (alepez) [reply examples/box_game/box_game_spectator.rs]: Yes, you're right. Fixed.

File: examples/box_game/box_game_spectator.rs (hunk 2 above)
Comment (alepez) [reply examples/box_game/box_game_spectator.rs]: Yes, you're right. Fixed.

File: examples/box_game/box_game_spectator.rs (hunk 1 above)
Comment (simbleau) [reply examples/box_game/box_game_spectator.rs]: Awesome!

**Comment by simbleau** [comment]
Should be good to merge @gschup

**Comment by gschup** [comment]
very nice! Thanks for the PR 💪

**Comment by alepez** [comment]
Do you think it is ready for a release?

**Comment by gschup** [comment]
I think it is time for another version, but I am currently travelling. Will probably do it in two weeks.

**Comment by donedgardo** [comment]
Awsome!

**Pull request merged**